Principles.md

```diff
@@ -44,6 +44,7 @@ Tactically, it can & maybe should cover:
 ### We act like owners and keep context
 * We love engineering and technology, but we solve for our customers and the bigger teams 1st.
 * If we can’t unanimously articulate why we’re doing something, we don’t do it.
+* If we have a nagging suspician we're headed in the wrong direction, we hit the Stop button, raise the issue, and rethink.
```

Comment:
`suspicion` is misspelled but otherwise this is a great addition. (I'm fine with the capitalized `Button` too)